FinalInAbstractClassRule complains not final method in interface

Starting with version 0.12 an check failed on an interface.

#### Steps required to reproduce the problem

```php
<?php
/**
 * This file is part of the ios-build package.
 *
 * Copyright (c) 2019, Thomas Mueller <mimmi20@live.de>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */

declare(strict_types = 1);
namespace IosBuild;

interface IosBuildInterface
{
    /**
     * Takes iOS build code and returns corresponding iOS version.
     *
     * @param string $needle the build code
     *
     * @throws BuildException
     * @throws NotFoundException
     *
     * @return string JSON encoded string
     */
    public function getVersion(string $needle): string;
}
```
2. run PHPStan including this library against this file.

#### Expected Result

```shell
 [OK] No errors
```

#### Actual Result

```shell
 ------ -------------------------------------------------------------------------------------------------------------------------
  Line   IosBuildInterface.php
 ------ -------------------------------------------------------------------------------------------------------------------------
  26     Method IosBuild\IosBuildInterface::getVersion() is not final, but since the containing class is abstract, it should be.
 ------ -------------------------------------------------------------------------------------------------------------------------


 [ERROR] Found 1 error

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,7 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## Unreleased
 
-For a full diff see [`0.12.1...master`](https://github.com/localheinz/phpstan-rules/compare/0.11.0...master).
+For a full diff see [`0.12.2...master`](https://github.com/localheinz/phpstan-rules/compare/0.12.2...master).
+
+## [`0.12.2`](https://github.com/localheinz/phpstan-rules/releases/tag/0.12.2)
+
+For a full diff see [`0.12.0...0.12.1`](https://github.com/localheinz/phpstan-rules/compare/0.12.1...0.12.2).
+
+### Fixed
+
+* Started ignoring interfaces from analysis by `Methods\FinalInAbstractClassRule` to avoid inappropriate errors ([#132](https://github.com/localheinz/phpstan-rules/pull/132)), by [@localheinz](https://github.com/localheinz)
 
 ## [`0.12.1`](https://github.com/localheinz/phpstan-rules/releases/tag/0.12.1)
 

--- a/src/Methods/FinalInAbstractClassRule.php
+++ b/src/Methods/FinalInAbstractClassRule.php
@@ -43,6 +43,10 @@ final class FinalInAbstractClassRule implements Rule
             return [];
         }
 
+        if ($containingClass->isInterface()) {
+            return [];
+        }
+
         if ($node->isAbstract()) {
             return [];
         }

--- a/test/Fixture/Methods/FinalInAbstractClassRule/Success/InterfaceWithPublicMethod.php
+++ b/test/Fixture/Methods/FinalInAbstractClassRule/Success/InterfaceWithPublicMethod.php
@@ -1,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Localheinz\PHPStan\Rules\Test\Fixture\Methods\FinalInAbstractClassRule\Success;
+
+interface InterfaceWithPublicMethod
+{
+    public function method(): void;
+}

--- a/test/Integration/Methods/FinalInAbstractClassRuleTest.php
+++ b/test/Integration/Methods/FinalInAbstractClassRuleTest.php
@@ -32,6 +32,7 @@ final class FinalInAbstractClassRuleTest extends AbstractTestCase
             'abstract-class-with-final-protected-method' => __DIR__ . '/../../Fixture/Methods/FinalInAbstractClassRule/Success/AbstractClassWithFinalProtectedMethod.php',
             'abstract-class-with-final-public-method' => __DIR__ . '/../../Fixture/Methods/FinalInAbstractClassRule/Success/AbstractClassWithFinalPublicMethod.php',
             'abstract-class-with-private-method' => __DIR__ . '/../../Fixture/Methods/FinalInAbstractClassRule/Success/AbstractClassWithPrivateMethod.php',
+            'interface-with-public-method' => __DIR__ . '/../../Fixture/Methods/FinalInAbstractClassRule/Success/InterfaceWithPublicMethod.php',
         ];
 
         foreach ($paths as $description => $path) {

